packages/block-library/src/legacy-widget/transforms.js

```diff
@@ -5,23 +5,23 @@ import { createBlock } from '@wordpress/blocks';
 
 const legacyWidgetTransforms = [
 	{
-		block: 'core/calendar',
-		widget: 'WP_Widget_Calendar',
```

Comment:
What is the context of renaming these?

Reply:
I don't think we ever should have had `class_name` in the REST API as:

1. The name that you give the PHP class which extends `WP_Widget` is an implementation detail, and;
2. Not all widgets extend `WP_Widget`.

Using `id_base` is the more correct way of identifying the kind of widget that a widget is.